TemplePlus/condition.cpp

```diff
@@ -5388,6 +5388,7 @@ int ItemCallbacks::UseableItemActionCheck(DispatcherCallbackArgs args){
 
 	if (objType == obj_t_scroll && !spellSys.CheckAbilityScoreReqForSpell(args.objHndCaller, spData.spellEnum, -1) && !useMagicDeviceSkillBase){
 		dispIo->returnVal = AEC_INVALID_ACTION;
+		logger->info("CheckAbilityScoreReqForSpell: Not enough ability score to use scroll!");
```

Comment:
Does this produce logspam? (being checked constantly)

Reply:
True. It will be checked many times for a scroll. But there is a problem of constant debugging of scrolls usage. Often it is unclear what was the problem of spell not firing. I had to add it. If it will live, I will submit probably more such logs. But not now, let's wait how it will present itself.

Reply:
I'm not sure I understand what you wrote, but at any rate log spam is not acceptable.

For your purposes I can offer changing this log message to logger->trace and adding a config setting for the log level so that it is not normally visible. (I'll add this config)